addons/website_sale/tests/test_sale_process.py

```diff
@@ -120,7 +120,7 @@ def test_04_admin_website_sale_tour(self):
             'show_line_subtotals_tax_selection': 'tax_excluded',
         }).execute()
 
-        self.start_tour("/", 'website_sale_tour_1')
+        self.start_tour("/", 'website_sale_tour_1', debug=True)
```

Comment:
🐞 
```suggestion
        self.start_tour("/", 'website_sale_tour_1')
```

Reply:
at least it prove that I double check :innocent: